kumascript/index.js

```diff
@@ -0,0 +1,31 @@
+const renderMacros = require("./src/render.js");
```

Comment:
I don't know what the exact best practices say but most other imports in this project use the import style *without* file extension. E.g https://github.com/mdn/stumptown-renderer/blob/master/content/index.js#L4
One immediate advantage I can think of is that we become unaffected if we change to TypeScript file extensions or something.